src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int DEFAULT_RADIUS = 10;
+    public static final int MAX_RANDOM_NUMBER = 100;
```

Comment:
public -> private,
be more attentive next time, I suggested this change in my previous review